draft-ietf-taps-interface.md

```diff
@@ -1113,9 +1113,13 @@ Type:
 
 Lifetime specifies how long a particular Message can wait to be sent to the
 remote endpoint before it is irrelevant and no longer needs to be
-(re-)transmitted. When a Message's Lifetime is infinite, it must be
-transmitted reliably. The type and units of Lifetime are
-implementation-specific.
+(re-)transmitted. This is a hint to the transport system -- it is not guaranteed
+that a Message will not be sent when its Lifetime has expired.
+
+Setting a Message's Lifetime to infinite indicates that the application does
+not wish to apply a time constraint on it, but it does not express a need for
```

Comment:
I find "a time constraint on it" confusing. Perhaps this should be "does not wish to apply a time constraint on the transmission of the message". An infinite lifetime does mean it should be transmitted at least once.

Reply:
I'm sorry, I merged this PR too quickly. I agree with your comment and will address it with a direct commit to the master.